media-libs/libgsf/libgsf-1.14.47.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY=" The G Structured File Library"
+DESCRIPTION=" The project aims to provide an efficient extensible i/o abstraction for \
+dealing with different structured file formats."
+HOMEPAGE="https://github.com/GNOME/libgsf"
+COPYRIGHT="2002-2020 Jody Goldberg"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/GNOME/libgsf/archive/LIBGSF_1_14_47.tar.gz"
+SOURCE_FILENAME="libgsf-$portVersion.tar.gz"
+SOURCE_DIR="libgsf-LIBGSF_1_14_47"
+CHECKSUM_SHA256="ca958db8b91113804e1c8ec772adb7cb3e44f81d404cf941330d47f3e25ccbe7"
+PATCHES=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
If this can't be build with gcc2 (!x86_gcc2) then it's best to add, commandSuffix, commandBinDir, --omit-dirs etc to the recipe where needed, check some of the existing recipes for this (I'm guessing you checked this on x86_64 so far and not the gcc2hybrid)

Reply:
That's correct, will look into this